_api-reference/nodes-apis/nodes-stats.md

```diff
@@ -977,6 +979,7 @@ active | Integer | The number of active threads in the pool.
 rejected | Integer | The number of tasks that have been rejected.
 largest | Integer | The peak number of threads in the pool.
 completed | Integer | The number of tasks completed.
+total_wait_time | Integer | The total time tasks spent waiting in the thread pool queue.
```

Comment:
Not all `thread_pool`s support this metric today -- only `search`, `search_throttled`, and `index_searcher` thread_pools support it. Not sure what the right place to mention this (if at all) is